### PR TITLE
Request for lead author review (now with the actual article)

### PR DESCRIPTION
**Pull request by m3mitsuppe**
I hope it works this time...

### CODE REVIEW

**Comment by abeggchr** [comment]
You can still edit. Just change and commit/push again. Ping me when you updated ther article.

**Comment by m3mitsuppe** [comment]
@abeggchr Editing finished - now really ready for lead author review

File: articles/paper-prototyping.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+2.	Create a set of wireframes, either directly on paper or using a tool like Balsamiq. Make sure to include all intermediate steps to your wireframes that a user would go through when performing the scenarios.
+3.	If one step involves adding a dialog on top of an existing page, create a wireframe "snippet" that just shows the dialog. This also works for other small changes on a screen.
+4.	The result is a stack of sketched or printed screens and a collection of snippets.
+
+![An paper prototype created using Balsamiq](./paper-prototyping/prototype-example.png)
+![An hand-sketched paper prototype](./paper-prototyping/paper-prototype-sketched.png)
+
+## Conducting the test
+
+Let "real" users of your system test the paper prototype. For a typical early-stage paper prototyping test, five representative users are enough to find the most important issues in your design.
```
Comment (peitor) [inline articles/paper-prototyping.md]: Why is "real" in quotes?  Not real users?

File: articles/paper-prototyping.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+4.	The result is a stack of sketched or printed screens and a collection of snippets.
+
+![An paper prototype created using Balsamiq](./paper-prototyping/prototype-example.png)
+![An hand-sketched paper prototype](./paper-prototyping/paper-prototype-sketched.png)
+
+## Conducting the test
+
+Let "real" users of your system test the paper prototype. For a typical early-stage paper prototyping test, five representative users are enough to find the most important issues in your design. 
+
+Your goal during the test is to observe users performing the scenarios. It's very much like a regular usability test where you replace the computer with a stack of paper. To prepare the test, write an instruction sheet that asks the users to perform the tasks underlying the scenarios you used for creating the wireframes.
```
Comment (peitor) [inline articles/paper-prototyping.md]: Hilft es da noch spezifischer zu werden? 
Observe over the shoulder? Influencing the user with body language? 
Wahrscheinlich geht das zu weit...

File: articles/paper-prototyping.md (hunk 1 above)
Comment (m3mitsuppe) [reply articles/paper-prototyping.md]: Good point. I'll remove the quotes.

File: articles/paper-prototyping.md (hunk 2 above)
Comment (m3mitsuppe) [reply articles/paper-prototyping.md]: I thought about this but decided to leave it as it is. Trying to stick to the main points here, of course there's more to know about paper prototyping than what I wrote about...

Comment (m3mitsuppe) [reply articles/paper-prototyping.md]: Thanks for the praise and the comments! I'm considering the article done for now.

**Pull request merged**